Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG BUILD_IMAGE
+ARG RUNTIME_IMAGE
+
+FROM $BUILD_IMAGE as builder
+
+ARG REDIS_VERSION
```

Comment:
I can't figure out what this does.

Reply:
Oh sorry that was one line left from testing the approach would work ok for Redis version support (the next PR)